src/transformers/models/bort/modeling_tf_bort.py

```diff
@@ -0,0 +1,250 @@
+    # See all BORT models at https://huggingface.co/models?filter=bort
+]
+
+BORT_START_DOCSTRING = r"""
+
+    This model inherits from :class:`~transformers.TFPreTrainedModel`. Check the superclass documentation for the
+    generic methods the library implements for all its model (such as downloading or saving, resizing the input
+    embeddings, pruning heads etc.)
+
+    This model is also a `tf.keras.Model <https://www.tensorflow.org/api_docs/python/tf/keras/Model>`__ subclass. Use
+    it as a regular TF 2.0 Keras Model and refer to the TF 2.0 documentation for all matter related to general usage
+    and behavior.
+
+    .. note::
+
+        TF 2.0 models accepts two formats as inputs:
+
+        - having all inputs as keyword arguments (like PyTorch models), or
+        - having all inputs as a list, tuple or dict in the first positional arguments.
+
+        This second option is useful when using :meth:`tf.keras.Model.fit` method which currently requires having all
+        the tensors in the first argument of the model call function: :obj:`model(inputs)`.
+
+        If you choose this second option, there are three possibilities you can use to gather all the input Tensors in
+        the first positional argument :
+
+        - a single Tensor with :obj:`input_ids` only and nothing else: :obj:`model(inputs_ids)`
+        - a list of varying length with one or several input Tensors IN THE ORDER given in the docstring:
+          :obj:`model([input_ids, attention_mask])` or :obj:`model([input_ids, attention_mask, token_type_ids])`
+        - a dictionary with one or several input Tensors associated to the input names given in the docstring:
+          :obj:`model({"input_ids": input_ids, "token_type_ids": token_type_ids})`
+
+    Parameters:
+        config (:class:`~transformers.BortConfig`): Model configuration class with all the parameters of the
+            model. Initializing with a config file does not load the weights associated with the model, only the
+            configuration. Check out the :meth:`~transformers.PreTrainedModel.from_pretrained` method to load the model
+            weights.
+"""
+
+
+@add_start_docstrings(
+    "The bare BORT Model transformer outputting raw hidden-states without any specific head on top.",
+    BORT_START_DOCSTRING,
+)
+class TFBortModel(TFBertModel):
+    """
+    This class overrides :class:`~transformers.TFBertModel`. Please check the superclass for the appropriate
+    documentation alongside usage examples.
+
+    Examples::
+        >>> from transformers import TFBortModel, BortTokenizer
+
+        >>> model = TFBortModel.from_pretrained("bort")
+        >>> tokenizer = BortTokenizer.from_pretrained("bort")
+
+        >>> inputs = tokenizer.encode_plus("The eastern endpoint of the canal is the Hubertusbrunnen.", return_tensors="tf")
+
+        >>> outputs = model(**inputs)
+        >>> hidden_states = outputs.last_hidden_state
+    """
+
+    config_class = BortConfig
+
```

Comment:
also add model type here for all models: https://github.com/huggingface/transformers/blob/51adb97cd644a5840d971868d18c1d436fd6ff5d/src/transformers/models/mt5/modeling_tf_mt5.py#L65